agent/src/agent/pipeline/config/stages/source/elastic.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from agent import source, monitoring
+from agent.pipeline.config.stages.base import JythonSource
+
+
+class ElasticScript(JythonSource):
+    JYTHON_SCRIPT = 'elastic_loader.py'
+
+    def _load_query_from_file(self):
+        with open(self.pipeline.config['query_file']) as f:
+            query = f.read()
+            return query.replace('${OFFSET}', '"OFFSET"')
```

Comment:
now we are not required to use ${OFFSET} so we can ask the client to use something json compatible like `$OFFSET` and you won't need to replace it two times